File: ABC/083/b.go
package main

import (
	"fmt"
)

func main() {
	var N, A, B int
	fmt.Scan(&N, &A, &B)

	result := 0
	for i := 1; i <= N; i++ {
		sum := sumDigits(i)

		if A <= sum && sum <= B {
			result += i
		}
	}

	fmt.Println(result)
}

func sumDigits(N int) (sum int) {
	for N != 0 {
		sum += N % 10
		N /= 10
	}

	return
}

File: ABC/083/b.py
N, A, B = map(int, input().split())


def digitSum(n):
    s = str(n)
    array = list(map(int, s))
    return sum(array)


result = 0
for i in range(N+1):
    s = digitSum(i)
    if A <= s and s <= B:
        result += i

print(result)
